<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExplorerIntroWidget</class>
 <widget class="QDockWidget" name="ExplorerIntroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="ExplorerIntro">
   <widget class="QGroupBox" name="ScenarioGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>181</width>
      <height>222</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="ShowcaseGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="ShowcaseComboBox">
          <property name="toolTipDuration">
           <number>-4</number>
          </property>
          <item>
           <property name="text">
            <string>Anaphylaxis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MultiTrauma</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LoadShowcase">
          <property name="text">
           <string>Load Showcase</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="PatientStateGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QComboBox" name="PatientStateComboBox">
          <item>
           <property name="text">
            <string>&lt;Patient States&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LoadPatientState">
          <property name="text">
           <string>Load State</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreatePatientButton">
       <property name="text">
        <string>New Patient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoadScenarioButton">
       <property name="text">
        <string>Load Scenario</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
